boards/arm/stm32l1_disco/doc/index.rst

```diff
@@ -0,0 +1,177 @@
+.. _stm32l1_disco_board:
+
+ST STM32LDISCOVERY and 32L152CDISCOVERY
+#######################################
+
+Overview
+********
+
+The two generations of the STM32L1 Discovery development boards come with
+an integrated the ST-LINK/V2 debugger and programmer.  The boards have a
+segment LCD and a touch slider, along with two user LEDs and a user button.
+Support circuitry for measuring power consumption is also available.
+It also comes with a comprehensive STM32 software HAL library and various
+packaged software examples.
+
+There
+are two variants of the board:
+
+- STM32LDISCOVERY targets STM32L152RBT6, with 128K flash, 16k ram
+- 32L152CDISCOVERY targets STM32L152RCT6, with 256K flash, 32K ram
```

Comment:
Please indicate which configuration is provided here.
Also, fyi, it's possible to provide variants of a board, cf `stm32min_dev`.

Reply:
I based the docs on that, but yeah, I suppose I could provide different dts files for them both.  I went with just a single one for the lower flash size, so it would work automatically on both.  Would that be enough, to just mention that?

Reply:
(that's what I've done for now)

Reply:
Ok, yes, please state it clearly.